src/python/qeopenfermion/_utils.py

```diff
@@ -232,7 +232,10 @@ def evaluate_operator_for_parameter_grid(ansatz, grid, backend, operator,
 	Returns:
 		value_estimate (zquantum.core.utils.ValueEstimate): stores the value of the expectation and its
 			 precision
+		optimal_parameters (numpy array): the ansatz parameters representing the ansatz parameters 
```

Comment:
I would add to the documentation that if there's more than one minimum, it returns only the first one found.

Reply:
Addressed here: `19e7b81`